pydash/pydash_app/dashboard/dashboard_repository.py

```diff
@@ -67,15 +58,15 @@ def seed_dashboards():
     to be used during development.
     """
 
-    # Clear current DB.
-    database_root.dashboards = MultiIndexedPersistentCollection({'id', 'url'})
+    # Clear current Dashboards-DB.
+    database_root.dashboards = MultiIndexedPersistentCollection({'id'})
+    _dev_dashboard_urls = ['http://pydash.io/', 'http://pystach.io/']
 
     # Fill in dashboards.
-    _development_dashboards = [
-        Dashboard(url='http://pydash.io/', user_id='000102030405060708090a0b0c0d0e0f'),
-        Dashboard(url='http://pistach.io/', user_id='f0e0d0c0b0a090807060504030201000')
-    ]
-    for dashboard in _development_dashboards:
-        print(f'Adding dashboard {dashboard}')
-        add(dashboard)
+    for user in database_root.users:
```

Comment:
Don't directly use the `database_root.users` here, because it is an implementation detail of the user repository. Instead, use `user_repository.all()` to fetch all users.